rskj-core/src/main/java/org/ethereum/rpc/dto/TransactionResultDTO.java

```diff
@@ -78,10 +78,10 @@ public TransactionResultDTO(Block b, Integer index, Transaction tx) {
         input = TypeConverter.toUnformattedJsonHex(tx.getData());
 
         if (!(tx instanceof RemascTransaction)) {
-            ECKey.ECDSASignature signature = tx.getSignature();
+            ECKey.ECDSASignature signature = tx.getSingleSignature();
```

Comment:
Should leave a TODO commit here indicating that this method does not support the multi-signature format.